aptos-move/framework/src/natives/aggregator_natives/aggregator_v2.rs

```diff
@@ -51,34 +54,29 @@ fn is_string_type(context: &SafeNativeContext, type_arg: &Type) -> SafeNativeRes
     Ok(false)
 }
 
-/// Given the list of native function arguments and a type, returns a tuple of its
-/// fields: (`aggregator id`, `limit`).
-pub fn pop_aggregator_fields_by_type(
-    ty_arg: &Type,
-    args: &mut VecDeque<Value>,
-) -> SafeNativeResult<(AggregatorID, u128)> {
+/// Given the list of native function arguments and a type, pop the next argument if it is of given type
+pub fn pop_value_by_type(ty_arg: &Type, args: &mut VecDeque<Value>) -> SafeNativeResult<u128> {
     match ty_arg {
-        Type::U128 => {
-            // Get aggregator information and a value to add.
-            let (id, limit) = aggregator_value_as_u128(&safely_pop_arg!(args, StructRef))?;
-            Ok((id, limit))
-        },
-        Type::U64 => {
-            // Get aggregator information and a value to add.
-            let (id, limit) = aggregator_value_as_u64(&safely_pop_arg!(args, StructRef))?;
-            Ok((id, limit as u128))
-        },
+        Type::U128 => Ok(safely_pop_arg!(args, u128)),
```

Comment:
Why is this change made? 
`aggregator_value_as_u128(&safely_pop_arg!(args, StructRef))?` and `safely_pop_arg!(args, u128)` does completely different things.